gearbox.go

```diff
@@ -151,7 +152,15 @@ func (gb *gearbox) newHTTPServer() *fasthttp.Server {
 
 // Stop serving
 func (gb *gearbox) Stop() error {
-	return gb.httpServer.Shutdown()
+	err := gb.httpServer.Shutdown()
+	if err != nil {
+		return err
+	}
+	if gb.address == "" {
```

Comment:
Make sure if it's not needed if shutdown will return error if address is not provided. Otherwise LGTM